synapse/rest/client/account.py

```diff
@@ -896,6 +896,36 @@ async def on_GET(self, request: SynapseRequest) -> Tuple[int, JsonDict]:
         return 200, response
 
 
+class AccountStatusRestServlet(RestServlet):
+    PATTERNS = client_patterns(
+        "/org.matrix.msc3720/account_status$", unstable=True, releases=()
+    )
+
+    def __init__(self, hs: "HomeServer"):
+        super().__init__()
+        self._auth = hs.get_auth()
+        self._store = hs.get_datastore()
+        self._is_mine = hs.is_mine
+        self._federation_client = hs.get_federation_client()
```

Comment:
these things all seem to be unused. Please don't import things from HomeServer where they are not required - it increases the coupling of the code.

Fixed in #12067.

Reply:
Gah, sorry about that. It came from a previous version of that implem that I refactored before opening the PR, but it looks like I forgot to remove some bits. Thanks for taking care of it!